tox.ini

```diff
@@ -10,7 +10,6 @@ requires =
 tox_pyenv_fallback = false
 
 [testenv]
-parallel_show_output = true
```

Comment:
`parallel_show_output` tells tox to always print all the output of every env that's run in parallel even if that job was successful. I don't think we want this: it just fills your screen with unnecessary output. With this removed we get the default behavior which is to print the full output of any envs that failed but not of ones that succeeded